modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -1289,7 +1289,13 @@ export class Wallet implements IWallet {
     common.validateParams(params, [], ['minFeeRate', 'changeAddressType', 'txFormat']);
     return this.bitgo
       .put(this.url())
-      .send({ minFeeRate: params.minFeeRate, changeAddressType: params.changeAddressType, txFormat: params.txFormat })
+      .send({
+        buildDefaults: {
```

Comment:
Isn't the arg type changed with this? Is it expected?

Reply:
Yes, the old arg type is not what `v2.wallet.update` was expecting